### PR TITLE
re: move `integration-tests/tests` to `integration-tests/src/tests`

### PR DESCRIPTION
**Pull request by pmnoxx**
By moving `integration-tests/tests` → `integration-tests/src/tests`. We can cut down the out of time it takes to run our CI.

### CODE REVIEW

File: integration-tests/src/tests/network/stress_network.rs [hunk 1]
```diff
@@ -70,13 +70,13 @@ fn make_peer_manager(seed: &str, port: u16, boot_nodes: Vec<(&str, u16)>) -> Pee
 /// The test is stopped gracefully (no panic) if some node other than node0 panicked.
 ///
 /// This was fixed in (#1954). To reproduce this bug:
-/// ```
+/// ```ignore
```
Comment (mina86) [inline integration-tests/src/tests/network/stress_network.rs:73]: This should be `shell` I think.

Comment (pmnoxx) [reply integration-tests/src/tests/network/stress_network.rs:73]: Thanks, fixed.

**Comment by mina86** [comment]
Have you checked Nayduck?

**Comment by pmnoxx** [comment]
https://nayduck.near.org/#/run/2306

**Comment by pmnoxx** [comment]
another try https://nayduck.near.org/#/run/2307

**Comment by pmnoxx** [comment]
It times like this I would like to have `./scripts/nayduck.py --verify`, a command that checks whenever all commands are valid, and whenever all test exist.

**Comment by pmnoxx** [comment]
@mina86 Would it be ok, if I add `--verify` option, which check whenever all test names are correct?
And paths exist?

**Comment by pmnoxx** [comment]
I'm making it into a draft until I write a script, which validates `nigthly.txt`.

File: nightly/expensive.txt [hunk 2]
```diff
@@ -29,7 +29,7 @@ expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving --features nightly_protocol,nightly_protocol_features
 
 expensive integration-tests test_catchup test_catchup
-expensive integration-tests test_catchup test_catchup --features nightly_protocol,nightly_protocol_features
+expensive integration-tests test_catchup tests::test_catchup::test_catchup --features nightly_protocol,nightly_protocol_features
```
Comment (mina86) [inline nightly/expensive.txt:32]: You will also need to change the third column from `test_catchup` to `integration_tests` I believe. Same in all other lines.

**Comment by mina86** [comment]
> I'm making it into a draft until I write a script, which validates nigthly.txt.

That’s essentially just running the commands locally. But if you want to improve `scripts/check_nightly.py` I’m in favour of that. I don’t see an easy way to get the names of the executables or Rust paths of test functions.

**Comment by pmnoxx** [comment]
> > I'm making it into a draft until I write a script, which validates nigthly.txt.
> 
> That’s essentially just running the commands locally. But if you want to improve `scripts/check_nightly.py` I’m in favour of that. I don’t see an easy way to get the names of the executables or Rust paths of test functions.

@mina86 
```
cargo test -p integration-tests -- --list
```

**Comment by pmnoxx** [comment]
@mina86 It should be good now.
I fixed one more test, which was missing.

https://nayduck.near.org/#/run/2311

Btw, I commented out ignored tests: 
- `test_seal_removal`
- `test_highload`

Those tests are not running, because of `#[ignore]`

File: nightly/expensive.txt [hunk 3]
```diff
@@ -28,8 +28,8 @@ expensive --timeout=1800 near-client near_client tests::catching_up::test_catchu
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving --features nightly_protocol,nightly_protocol_features
 
-expensive integration-tests test_catchup test_catchup
-expensive integration-tests test_catchup test_catchup --features nightly_protocol,nightly_protocol_features
+expensive integration-tests test_catchup tests::test_catchup::test_catchup
```
Comment (mina86) [inline nightly/expensive.txt:31]: I’m pretty sure you need to change the third value from `test_catchup` to `integration_tests`. Similarly throughout the whole file.

File: nightly/expensive.txt [hunk 4]
```diff
@@ -56,65 +56,66 @@ expensive --timeout=500 near-client near_client tests::consensus::test_consensus
 expensive --timeout=500 near-client near_client tests::consensus::test_consensus_with_epoch_switches --features nightly_protocol,nightly_protocol_features
 
 # testnet rpc
-expensive nearcore test_tps_regression test::test_highload
-expensive nearcore test_tps_regression test::test_highload --features nightly_protocol,nightly_protocol_features
+# Is disabled, because it has `#[ignore]`
+# expensive nearcore test_tps_regression test::test_highload
+# expensive nearcore test_tps_regression test::test_highload --features nightly_protocol,nightly_protocol_features
```
Comment (mina86) [inline nightly/expensive.txt:61]: For the purposes of NayDuck ignored tests are passing so this should stay enabled. Alternatively, disable this test in NayDuck but then remove `#[ignore]`. Best to disable tests in single location. Same for `test_seal_removal`.

Comment (pmnoxx) [reply nightly/expensive.txt:61]: @mina86 Does nayduck run all tests with `#[ignore]`?

Comment (mina86) [reply nightly/expensive.txt:61]: NayDuck runs what it’s configured to run and yes, all `#[ignore]` tests are currently listed.

Comment (pmnoxx) [reply nightly/expensive.txt]: @mina86 Ok, I undid this change. I'll assume it's ok to have here tests with `#[ignore]`.

File: nightly/expensive.txt (hunk 3 above)
Comment (pmnoxx) [reply nightly/expensive.txt:31]: @mina86 Ok, I fixed it.

**Comment by pmnoxx** [comment]
https://nayduck.near.org/#/run/2312

File: nightly/expensive.txt [hunk 5]
```diff
@@ -56,66 +56,65 @@ expensive --timeout=500 near-client near_client tests::consensus::test_consensus
...
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_simple_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_with_args_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_with_args_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_swap_key_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_swap_key_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_upload_contract_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_upload_contract_testnet --features nightly_protocol,nightly_protocol_features
+expensive integration-tests integration-tests tests::standard_cases::rpc::test::test_access_key_smart_contract_testnet
```
Comment (mina86) [inline nightly/expensive.txt:62]: It should be `integration_tests` with underscore I think.

**Comment by pmnoxx** [comment]
https://nayduck.near.org/#/run/2313

File: nightly/expensive.txt (hunk 5 above)
Comment (pmnoxx) [reply nightly/expensive.txt]: @mina86 I see, so the 3rd argument is the file name.

Comment (mina86) [reply nightly/expensive.txt]: Yep. As documented in `nightly/README.md`.

**Pull request closed without merging**